pkg/server/plugin/upstreamca/awssecret/awssecret_test.go

```diff
@@ -81,34 +79,22 @@ func (as *AWSSecretSuite) TestGetSecretFail() {
 }
 
 func (as *AWSSecretSuite) Test_SubmitValidCSR() {
-	const testDataDir = "_test_data/csr_valid"
-	validCsrFiles, err := ioutil.ReadDir(testDataDir)
+	validSpiffeID := "spiffe://localhost"
+	csr, err := util.NewCSRTemplate(validSpiffeID)
 	as.Require().NoError(err)
 
-	for _, validCsrFile := range validCsrFiles {
-		csrPEM, err := ioutil.ReadFile(filepath.Join(testDataDir, validCsrFile.Name()))
-		as.Require().NoError(err)
-		block, rest := pem.Decode(csrPEM)
-		as.Require().Len(rest, 0)
-
-		resp, err := as.awsUpstreamCA.SubmitCSR(ctx, &upstreamca.SubmitCSRRequest{Csr: block.Bytes})
-		as.Require().NoError(err)
-		as.Require().NotNil(resp)
-	}
+	resp, err := as.awsUpstreamCA.SubmitCSR(ctx, &upstreamca.SubmitCSRRequest{Csr: csr})
+	as.Require().NoError(err)
+	as.Require().NotNil(resp)
 }
 
 func (as *AWSSecretSuite) Test_SubmitInvalidCSR() {
```

Comment:
Please add a test case for an invalid sequence of bytes.

Reply:
I added a new test case with https://github.com/spiffe/spire/pull/988/commits/aa8cfcb3200e51553c35c567ee56c85b41f02140 .
Is this the test you expected?